### ISSUE TITLE
Clarification regarding duplicate bugs

### ISSUE THREAD

**Issue by Punpun1643** [nus-cs2103-AY2122S2/forum#254]
On the module website, it is stated that bugs are considered as duplicates if the behaviours are **caused by the same defect and cannot be fixed independently**. As per the screenshot below:

<img width="738" alt="Screenshot 2022-04-04 at 9 32 35 PM" src="https://user-images.githubusercontent.com/60144099/161555164-7da51c0e-df7e-4af0-8105-5e9973030c85.png">

Based on this, I am still not quite sure what "cannot be fixed independently" means. Does it mean that if I fix bug A, bug B would be fixed as well, hence bug A and bug B cannot be fixed independently, therefore duplicate?  

In a scenario where, for example, a bug is actually the result of some problems due to the `INDEX` which would affect commands that involve the use of `INDEX` e.g. `delete INDEX`, `view INDEX` etc. Would the report **delete command does not delete the correct INDEX** **and view command does not view the correct INDEX** be considered as duplicate?

**Comment by damithc** [comment]
> Based on this, I am still not quite sure what "cannot be fixed independently" means. Does it mean that if I fix bug A, bug B would be fixed as well, hence bug A and bug B cannot be fixed independently, therefore duplicate?

@Punpun1643 That's correct. This is because if they are really duplicates, fixing one of them should automatically fix all of them. But note that if the same cause is resulting in buggy behavior in many areas, while they can be considered duplicates, the severity should be selected/adjusted appropriately to match the high impact of the bug.

**Comment by Punpun1643** [comment]
@damithc Alright prof! Thank you for the clarification!
